cmd/root.go

```diff
@@ -68,11 +73,37 @@ func New(configOptions *ConfigOptions) *cobra.Command {
 
 // Execute creates the root command and runs it.
 func Execute() {
-	if err := New(nil).Execute(); err != nil {
+	ctx := WithSignals(context.Background())
+	if err := New(nil).ExecuteContext(ctx); err != nil {
 		logger.WithError(err).Fatal("error executing event-generator")
 	}
 }
 
+// WithSignals returns a copy of ctx  a new context.
```

Comment:
The doc here is not very clear. Probably a typo or something

Reply:
Good catch!
Fixing it